flake.nix

```diff
@@ -11,14 +11,15 @@
   inputs = {
     flake-compat.flake = false;
     naersk.url = "github:nix-community/naersk";
+    nixos-nixpkgs.url = "github:nixos/nixpkgs";
```

Comment:
I was rather suggesting using something like this
```suggestion
    nixos-nixpkgs.url = "github:nixos/nix?ref=2.21.4";
```

Reply:
Fixed